service/agentconf.go

```diff
@@ -227,13 +227,7 @@ func (s *systemdServiceManager) WriteSystemdAgents(
 
 		// Otherwise we need to manually ensure the service unit links.
 		svcFileName := svcName + ".service"
-		if err = os.Symlink(path.Join(systemd.LibSystemdDir, svcName, svcFileName),
```

Comment:
This method (WriteSystemdAgents) is all at sea. I will refactor it in a forthcoming patch.